tests/unit/testUncaught.js

```diff
@@ -50,6 +52,13 @@ test('Uncaught handler setup', function (t) {
 });
 
 test('Test uncaught shutdown behavior', function (t) {
+  if (!isString(process.env.GOOGLE_APPLICATION_CREDENTIALS)
```

Comment:
@matthewloring on second look - the reason this is required is becuase the second part of this test utilizes the google-auth-library which needs `GOOGLE_APPLICATION_CREDENTIALS` set (or some other valid form of configuration) so that it won't throw